src/infra/auth/api-keys.ts

```diff
@@ -27,3 +29,42 @@ export const retrieveAPIKey = async (
 		await permissions.authorizationMiddleware(req);
 	}
 };
+
+export const isApiKeyOfActorType = async (
```

Comment:
This will replace a pre-existing part of the cloud API.

Reply:
`infra` is for bits that would in theory move into pinejs or a pinejs module - it shouldn't have anything balena-specific